### ISSUE TITLE
Cannot log in to certain worlds/world switch not functioning

### ISSUE THREAD

**Issue by LegendArtorias** [runelite/runelite#12234]
I've been having troubles lately trying to log in to certain worlds (I'd say as much as 30% of worlds, too many for me to manually keep track of or have the time to write them all down) and even using the world hop feature of vanilla OSRS through RuneLite. To start with, the *click to switch* button on the login screen does not load at all when attempting to switch worlds that way. The *world hopper* option ingame does not load at all either (the default vanilla OSRS one). The only way I can switch worlds is through the RuneLite world hopper tool. To make matters worse, certain worlds cannot be logged into at all. It just attempts to log in to that world, sits for about 30 seconds and then boots me back to the login screen telling me that there was an error connecting to the world. This did not happen immediately with RuneLite and I believe it to be a setting/tool that I changed, but I cannot for the life of me track down what it is. Any help would be appreciated as I have to sit and wait for the login to fail every time I try a new world and it is very time-consuming.

Steps to reproduce the bug:
Attempt to hop to any world via the RuneLite world hopper
Be kicked to the login screen

Expected behavior:
Attempt to hop to any world via the RuneLite world hopper or default OSRS world hopper (which should ideally be working)
Switch worlds successfully

Windows 10
RuneLite version: 1.6.23
Launcher version: 2.1.3

**Comment by Nightfirecat** [comment]
This does not sound like a RuneLite issue. I would expect the same behavior to be present in the vanilla client.

**Comment by LegendArtorias** [comment]
Thanks for the response. I have been having trouble with my internet provider and accessing certain sites as of late, however since I could log in to some worlds and not others I did not think this to be the case. But just in case, I contacted my service provider and had them troubleshoot the problem and resolve it. After that, everything seems to be working perfectly. Thank you very much, that was all the feedback I needed. Problem solved :)
